kube-burner/hcp-density-aks/templates/deployment.yaml

```diff
@@ -0,0 +1,47 @@
+---
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: {{.name}}-{{.Replica}}
+spec:
+  replicas: {{.podReplicas}}
+  selector:
+    matchLabels:
+      app: {{.name}}-{{.Replica}}
+      svc: {{.name}}-{{.Replica}}
+  template:
+    metadata:
+      labels:
+        app: {{.name}}-{{.Replica}}
+        svc: {{.name}}-{{.Replica}}
+    spec:
+      nodeSelector:
+        node-role.kubernetes.io/worker: ""
+      containers:
+      - args:
+        - sleep
+        - infinity
+        image: registry.k8s.io/pause:3.1
+        resources:
+          requests:
+            cpu: 10m
+            memory: 10M
```

Comment:
These requests have to be changed according to [Grafana URL](https://grafana.rdu2.scalelab.redhat.com:3000/d/SNoAlkE4k/hypershift-report-multi-hc?orgId=1&from=1706890903943&to=1706895895604&var-Datasource=AWS+Pro+-+ripsaw-kube-burner&var-mgmt_cluster_name=aromc1-djt22&var-workload=cluster-density-v2&var-svc_cluster_name=&var-latencyPercentile=P99&var-nonServingWorker=All&var-servingWorker=All&var-uuid=80f4401f-4be0-49e3-9757-d6db4b7968ec&var-hosted_cluster_name=aro-hosted-cp-7wncl)